be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/ElectionHandler.scala

```diff
@@ -41,25 +40,25 @@ class ElectionHandler(dbRef: => AskableActorRef) extends MessageHandler {
   /** Overrides default DbActor with provided parameter
     */
   override final val dbActor: AskableActorRef = dbRef
+  private val serverUnexpectedAnswer: String = "The server is doing something unexpected"
```

Comment:
I do not see where you use this defined value. It seems that you do not use it at all. Correct me if I am wrong.

Reply:
Oops my bad, it is all right !